lms/envs/common.py

```diff
@@ -51,11 +51,6 @@
 
 PLATFORM_FACEBOOK_ACCOUNT = "http://www.facebook.com/YourPlatformFacebookAccount"
 PLATFORM_TWITTER_ACCOUNT = "@YourPlatformTwitterAccount"
-PLATFORM_TWITTER_URL = "https://twitter.com/YourPlatformTwitterAccount"
```

Comment:
Hmm, I can't seem to find them when I grep the code base.  (I'm finding the `_ACCOUNT` settings, which aren't deleted, but not the `_URL` settings).